src/plugins/menu_qrcode.py

```diff
@@ -39,6 +44,7 @@ def drawrect(self, row, col):
 
 class QRCodeDialog(QtGui.QDialog):
     """The dialog"""
+
```

Comment:
?

Reply:
Just give 1 blank line after class docstring, for good readability

Reply:
It's a matter of taste. I don't think the blank line improves readability. And you didn't add extra blank lines in the `network` package (#1518). So what's your real reason, to make more changes?